nep-25.mediawiki

```diff
@@ -0,0 +1,100 @@
+<pre>
+  NEP: 25
```

Comment:
It can't be, we've reserved NEP 25 for https://github.com/neo-project/proposals/pull/160. And these two should be NEP 26 and NEP 27. Otherwise we need to adjust the https://github.com/neo-project/proposals/pull/160.

Reply:
The number system is trash, we should use the pr number, instead of a number that can be reserved here and there. @shargon @roman-khimov @NGDAdmin @AnnaShaleva @vncoelho @cschuchardt88

Reply:
Or at least someone should manage the number, and draft should be assigend an NEP number and should use the given NEP number, even if the draft is not adopted, the number should be consdierd used.

Reply:
I agree, and that's want it should be the `PR number` not some made up number.

Reply:
This is blockchain tech, get with the picture guys, I'm sick of do what you think. Let's do with what works.

Reply:
> The number system is trash

I don't think we have a problem with numbers.